### PR TITLE
Ability to specify the d3 defined prop in Line-, LineMark- and AreaSeries.

### PR DESCRIPTION
**Pull request by tijp**
A PR that enables the ability the specify a defined prop in the Line-, LineMark- and AreaSeries based on this issue https://github.com/uber/react-vis/issues/457.

This let's users create series containing null (or undefined) data where the null values are not automatically converted to 0's, but can be ignored in the drawing process like in the image below.

`<LineSeries defined={(d) => d.y !== null} data={data} />`
27289292-b70eb0ec-5509-11e7-88b1-8791512dc0e3

### CODE REVIEW

**Comment by tijp** [comment]
Thanks for the great feedback!
I made the requested changes. If you could review those that would be great!

File: src/plot/series/line-series.js [hunk 1]
```diff
@@ -113,7 +96,22 @@ class LineSeries extends AbstractSeries {
 }
 
 LineSeries.displayName = 'LineSeries';
-LineSeries.defaultProps = defaultProps;
-LineSeries.propTypes = propTypes;
+LineSeries.propTypes = {
```
Comment (mcnuttandrew) [inline src/plot/series/line-series.js]: plus one for good reorganization

File: tests/components/line-series-tests.js [hunk 2]
```diff
@@ -104,3 +105,43 @@ test('LineSeries: Line Styling', t => {
   t.equal(lineStyle.strokeDasharray, '3, 1', 'should honor stroke dash-array');
   t.end();
 });
+
+test('NullAccessor prop: Showcase Example - Null Data Example', t => {
```
Comment (mcnuttandrew) [inline tests/components/line-series-tests.js]: 👏 good test👏

**Comment by woniesong92** [comment]
@tijp Thanks for making this PR. I ran into this exact use case and it saved me a ton of time.

**Comment by girlslearningcodeyul** [comment]
@tijp Thank you for this. I am not able to see and implement defined={} as a prop of LineSeries. Is this functionality still in place? Thank you

**Comment by NeroVanbiervliet** [comment]
the property has been renamed to `getNull`

**Pull request merged**